Source/core/Portability.h

```diff
@@ -396,7 +396,6 @@ inline void EXTERNAL SleepS(unsigned int a_Time)
 #endif
 
 #if defined(_THUNDER_DEBUG) || !defined(_THUNDER_NDEBUG)
-#define __DEBUG__
```

Comment:
As discussed on slack this is an internal flag that is now exposed. Just use the external flags.